src/zui/ZUIOrganizeSidebar/SidebarListItem.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/display-name */
```

Comment:
why does thos component require this rule to be disabled?

Reply:
Because it uses `forwardRef`. There is a way around this which is to use a regular `function` with a name, but I don't think that is necessary in this case (even though I do think it's an easy fix that I would have liked).

Reply:
I'll do it :+1:

Reply:
Another way to fix it is to assign a property to it. I went with that solution.